hips/hip-0019.md

```diff
@@ -0,0 +1,63 @@
+--- 
+hip: 0019
+title: "Add Exclude File Option to Helm Lint Command"
+authors: Danilo Patrucco
+created: "2024-07-03"
+type: "feature"
+status: "draft"
+---
+
+## Abstract
+This proposal introduces enhancements to the `helm lint` command, allowing the exclusion of specific files or patterns through a `.helmlintconfig.yaml` file, analogous to `.gitignore` or `.helmignore`. Additionally, a `--lint-config-file` flag will be added to specify alternative locations for the `.helmlintconfig.yaml` file. This feature is particularly useful in projects with multiple sub-charts. An environment variable `HELM_LINT_CONFIG_FILE` will also be introduced to mirror the functionality of the flag.
```

Comment:
```suggestion
This proposal introduces enhancements to the `helm lint` command, allowing the exclusion of specific files or patterns through a `.helmlintignore` file, analogous to `.gitignore` or `.helmignore`. Additionally, a `--lint-config-file` flag will be added to specify alternative locations for the `.helmlintignore` file. This feature is particularly useful in projects with multiple sub-charts. An environment variable `HELM_LINT_IGNORE_FILE` will also be introduced to mirror the functionality of the flag.
```

Reply:
Also, We did put the PR back in Draft, since we are waiting to have a finalized version of the HIP to continue the development work on the helm repo.  Please let me know If you have any additional reservation, and thank you for the review